artifactory/utils/container/image.go

```diff
@@ -9,45 +9,56 @@ import (
 )
 
 type Image struct {
-	tag string
+	// Image name includes the registry domain and image name and image tag.
```

Comment:
// Image name includes the registry domain and image name and image tag.
-->
// Image name includes the registry domain, image name and image tag.